handlers/sower.go

```diff
@@ -47,6 +48,12 @@ func dispatch(w http.ResponseWriter, r *http.Request) {
 
 	var currentAction = inputRequest.Action
 
+	var accessFormat string = "presigned-url"
```

Comment:
can u change presigned-url to presigned_url? since access_format uses an underscore